schemas/2015-05-21-preview/Microsoft.DevTestLabs.json

```diff
@@ -0,0 +1,59 @@
+{
+  "id": "http://schema.management.azure.com/schemas/2015-08-01/Microsoft.DevTestLab.json#",
```

Comment:
Please align the schema id with the schema file path, (use "2015-05-21-preview" at both places.)